addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -155,7 +155,7 @@
                         </group>
                     </group>
                     <h4 attrs="{'invisible': [('raw_workorder_line_ids', '=', [])]}">Components</h4>
-                    <field name="raw_workorder_line_ids" attrs="{'invisible': [('raw_workorder_line_ids', '=', [])], 'readonly': [('state', 'in', 'cancel', 'done')]}">
+                    <field name="raw_workorder_line_ids" attrs="{'invisible': [('raw_workorder_line_ids', '=', [])], 'readonly': [('state', 'in', ('cancel', 'done'))]}">
```

Comment:
will this commit of "various fixes in xml views" be backported (full or partially)? 🤞

Reply:
@mreficent some of these fixes have or will be backported, but not systematically.